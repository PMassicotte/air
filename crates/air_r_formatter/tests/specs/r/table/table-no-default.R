#' [format]
#' table = ["foo"]
#' default-table = false

# Should not format as table
tribble(
~x,~y,
1,2,
3,4
)

# Should not format as table
fcase(
x<5L,1L,
x>5L,3L,
default=5L
)

foo(
~x,~y,
1,2,
3,4
)
