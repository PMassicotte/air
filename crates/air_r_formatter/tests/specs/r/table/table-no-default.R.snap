---
source: crates/air_formatter_test/src/snapshot_builder.rs
info: r/table/table-no-default.R
---
# Input

```R
#' [format]
#' table = ["foo"]
#' default-table = false

# Should not format as table
tribble(
~x,~y,
1,2,
3,4
)

# Should not format as table
fcase(
x<5L,1L,
x>5L,3L,
default=5L
)

foo(
~x,~y,
1,2,
3,4
)

```


=============================

# Outputs

## Output 1

-----
Indent style: Space
Indent width: 2
Line ending: LF
Line width: 80
Persistent line breaks: Respect
Table: foo
-----

```R
#' [format]
#' table = ["foo"]
#' default-table = false

# Should not format as table
tribble(
  ~x,
  ~y,
  1,
  2,
  3,
  4
)

# Should not format as table
fcase(
  x < 5L,
  1L,
  x > 5L,
  3L,
  default = 5L
)

foo(
  ~x , ~y ,
   1 ,  2 ,
   3 ,  4
)
```
